README.md

```diff
@@ -191,6 +191,7 @@ Beschrijving van de omzetstappen en aanvullende inputvelden:
 **Let op:**
 - Wanneer de gebruiker beschikt over een access database, raden wij aan `digitale_standaard` omzetting te gebruiken, ook als de shapefile alle informatie bevat. Hierbij is de kans op handmatige fouten kleiner.
 - Velden die beginnen met `EDIT` kunnen door de gebruiker worden aangepast en hebben effect op de vervolgstappen van veg2hab. Velden die beginnen met `INTERN` zijn boekhoudvelden die veg2hab nodig heeft, en mogen niet door de gebruiker worden aangepast. Overige velden kunnen door de gebruiker veranderd worden, maar dit heeft geen effect op veg2hab.
+- ArcGIS Pro (en veg2hab) kan niet goed omgaan met velden die beginnen met een getal of speciaal teken. Als inladen van een vegetatiekartering met stap 1 niet goed lukt, controleer dan of de gebruikte velden met een letter beginnen. Zo niet, pas dit dan aan, of kopieer de informatie naar nieuwe velden die beginnen met een letter. 
```

Comment:
Het hernoemen is lastiger dan ik dacht; als je het via rechtermuis op een kolomnaam in de attribute table > field doet kun je enkel de alias aanpassen (hier ging het bij jou ook fout denk ik). Je moet dus of een nieuwe kolom maken, of de kolom hernoemen via Geoprocessing/Tools > Data Management Tools > Fields > Alter Field (https://pro.arcgis.com/en/pro-app/latest/tool-reference/data-management/alter-field-properties.htm). Misschien handig om daar goed expliciet over te zijn hier voor we support vragen krijgen. (want mensen lezen natuurlijk van tevoren sowieso goed de handleiding ;)